source/adios2/engine/bp5/BP5Reader.cpp

```diff
@@ -496,6 +493,8 @@ void BP5Reader::PerformLocalGets()
             m_BP5Deserializer->FinalizeGet(Req, false);
         }
     }
+    m_BP5Deserializer->FinalizeDerivedGets(ReadRequests);
```

Comment:
Is this supposed to be included between macros `#ifdef ADIOS2_HAVE_DERIVED_VARIABLE`

Reply:
FinalizeDerivedGets is defined without or without derived vars, it just has a null body without them.  Could go either way here, but having a null body keeps the number of #ifdef's down at least a little bit.